cloud/ec2.go

```diff
@@ -195,9 +195,9 @@ const (
 )
 
 const (
-	checkSuccessAttempts   = 12
+	checkSuccessAttempts   = 10
```

Comment:
This is not a problem, but for my own curiosity, why did you choose to reduce it from 12 to 10?

Reply:
It's just a small adjustment to reduce the number of retries delayed by 1 minute. I calculated the timing and with 10 attempts and an exponential backoff maxing out at a max delay of 1 minute, and it'll have five 1-minute check intervals. Preferably, we would check a little more often than 1 minute since it's important to not let the user wait a long time for their host to be up, so I'm relying on the fact that the job will retry as well to restart the exponential backoff loop again. The combined retry mechanisms should overall provide more retrying (but with less waiting between each check).